ipfix/node.c

```diff
@@ -235,7 +243,7 @@ static u8* format_netflow_v10_data_packet(u8 *s, va_list *args) {
           s = format(s, "\t\t%U", format_ip4_address, data);
           break;
         case protocolIdentifier:
-          s = format(s, "\t\t%u", ntohl(*(u16 *)data));
+          s = format(s, "\t\t%u", *(u8 *)data);
```

Comment:
Is this host or network endlessness? Just checking this isn't suppose to be `clib_byte_swap_u8(*(u8 *)data)`.

Reply:
It's network, but since endianness is at byte granularity here (there's no `clib_byte_swap_u8` function) and the data is only one byte I think it's ok.